features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/metric/interactiontonextview/TimeBasedInteractionIdentifierTest.kt

```diff
@@ -117,5 +117,30 @@ internal class TimeBasedInteractionIdentifierTest : ObjectTest<TimeBasedInteract
         assertThat(result).isFalse()
     }
 
+    @Test
+    fun `M return true W defaultThresholdUsed{default delay used}`() {
+        // Given
+        val testedValidator = TimeBasedInteractionIdentifier()
+
+        // When
+        val result = testedValidator.defaultThresholdUsed()
+
+        // Then
+        assertThat(result).isTrue()
+    }
+
+    @Test
+    fun `M return false W defaultThresholdUsed{custom delay used}`(forge: Forge) {
+        // Given
+        val testedValidator = TimeBasedInteractionIdentifier(
+            TimeBasedInteractionIdentifier.DEFAULT_TIME_THRESHOLD_MS + forge.aLong(min = 0, max = 100)
```

Comment:
minor: `forge.aLong(min = 0, max = 100)` can be replaced by the `@LongForgery(min = 0, max = 100)` test method argument.